apps/web/src/pages/templates/components/TriggerSnippetTabs.tsx

```diff
@@ -61,17 +66,19 @@ novu.trigger('${identifier}', ${JSON.stringify(
 
 export const getCurlTriggerSnippet = (
   identifier: string,
-  variables: INotificationTriggerVariable[],
-  subscriberVariables: INotificationTriggerVariable[]
+  to: Record<string, any>,
+  payload: Record<string, any>,
+  overrides?: Record<string, any>
 ) => {
   const curlSnippet = `curl --location --request POST '${API_ROOT}/v1/events/trigger' \\
      --header 'Authorization: ApiKey <REPLACE_WITH_API_KEY>' \\
      --header 'Content-Type: application/json' \\
      --data-raw '${JSON.stringify(
        {
          name: identifier,
-         to: { ...getSubscriberValue(subscriberVariables, (variable) => variable.value || '<REPLACE_WITH_DATA>') },
-         payload: { ...getPayloadValue(variables) },
+         to,
+         payload,
+         overrides,
```

Comment:
Can we only show the overrides object here if it contains any keys?

Reply:
Because of the JSON.stringify it won't show the overrides if it is an empty object